src/layers.c

```diff
@@ -469,17 +469,18 @@ handle_new_popup(struct wl_listener *listener, void *data)
 	wlr_output_layout_get_box(server->output_layout,
 		output->wlr_output, &output_box);
 
+	struct wlr_box usable = output_usable_area_in_layout_coords(output);
```

Comment:
`struct wlr_box output_box` can be removed.

Reply:
And `struct server *server` too